recipes/sctagger/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+mkdir -p ${PREFIX}/bin
+cp -f scTagger.py ${PREFIX}/bin 
```

Comment:
I think it might be worth adding a setup.py file, so you would not need this shell script anymore.

```suggestion

```

Reply:
@mirand863 do you an example repo that I can follow? I can create a new version with the `setup.py` file.

Reply:
> @mirand863 do you an example repo that I can follow? I can create a new version with the `setup.py` file.

I cannot remember a good example, but I find this tutorial very enlightening https://circleci.com/blog/publishing-a-python-package/
If you really must have an example, I can look for one here on bioconda.

Reply:
@baraaorabi ok, so I took a quick look and [pyfaidx](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pyfaidx/meta.yaml) seems to be a good example. Ultimately, what you want is to be able to run pip install in the conda recipe to install your script. Also, might be a good idea to rename without the .py extension.